internal/cmd/users.go

```diff
@@ -138,6 +140,7 @@ func newUsersListCmd(cli *CLI) *cobra.Command {
 					rows = append(rows, row{
 						Name:       user.Name,
 						LastSeenAt: HumanTime(user.LastSeenAt.Time(), "never"),
+						Providers:  strings.Join(user.ProviderNames, ", "),
```

Comment:
Does this include the `Infra` provider? We should filter that out

Reply:
I think that kills a lot of the usefulness of this feature. It's super useful to see where the users are coming from, and you wouldn't be able to tell that the user was in both an IDP and Infra without it.